lib/clean_architecture/serializers/success_collection_payload.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'duckface'
+require 'clean_architecture/interfaces/success_payload'
+require 'clean_architecture/serializers/success_object_payload'
+
+module CleanArchitecture
+  module Serializers
+    class SuccessCollectionPayload
+      implements_interface CleanArchitecture::Interfaces::SuccessPayload
+
+      attr_reader :version
```

Comment:
We have both `attr_reader :version` and `def version`, I guess thats not intended?

Reply:
Correct, thanks for the catch. Removing.